pom.xml

```diff
@@ -691,43 +728,26 @@
                     <version>2.9</version>
                 </plugin>
 
-                <plugin>
-                    <groupId>org.codehaus.mojo</groupId>
-                    <artifactId>animal-sniffer-maven-plugin</artifactId>
-                    <version>1.15</version>
-                    <executions>
-                        <execution>
-                            <id>check-jdk6</id>
-                            <phase>process-classes</phase>
-                            <goals>
-                                <goal>check</goal>
-                            </goals>
-                            <configuration>
-                                <signature>
-                                    <groupId>org.codehaus.mojo.signature</groupId>
-                                    <artifactId>java16</artifactId>
-                                    <version>1.0</version>
-                                </signature>
-                                <annotations>
-                                    <annotation>com.datastax.driver.core.IgnoreJDK6Requirement</annotation>
-                                </annotations>
-                            </configuration>
-                        </execution>
-                        <execution>
-                            <id>check-jdk8</id>
-                            <goals>
-                                <goal>check</goal>
-                            </goals>
-                            <configuration>
-                                <signature>
-                                    <groupId>org.codehaus.mojo.signature</groupId>
-                                    <artifactId>java18</artifactId>
-                                    <version>1.0</version>
-                                </signature>
-                            </configuration>
-                        </execution>
-                    </executions>
-                </plugin>
+<!--                <plugin>-->
+<!--                    <groupId>org.codehaus.mojo</groupId>-->
+<!--                    <artifactId>animal-sniffer-maven-plugin</artifactId>-->
+<!--                    <version>1.15</version>-->
+<!--                    <executions>-->
+<!--                        <execution>-->
+<!--                            <id>check-jdk8</id>-->
+<!--                            <goals>-->
+<!--                                <goal>check</goal>-->
+<!--                            </goals>-->
+<!--                            <configuration>-->
+<!--                                <signature>-->
+<!--                                    <groupId>org.codehaus.mojo.signature</groupId>-->
+<!--                                    <artifactId>java18</artifactId>-->
+<!--                                    <version>1.0</version>-->
+<!--                                </signature>-->
```

Comment:
I believe you could have excluded the OpenTelemetry dependency instead of disabling the entire check:

```
<configuration>
    ...
    <excludeDependencies>
        <excludeDependency>io.opentelemetry:*</excludeDependency>
    </excludeDependencies>
 </configuration>
```